app/controllers/activitypub/replies_controller.rb

```diff
@@ -38,7 +38,7 @@ def replies_collection_presenter
       type: :unordered,
       part_of: account_status_replies_url(@account, @status),
       next: next_page,
-      items: @replies.map { |status| status.local ? status : status.id }
+      items: @replies.map { |status| status.local ? status : status.uri }
```

Comment:
For consistency shouldn't this use TagManager?

Reply:
I'm not sure I see the use for that. I don't think calling TagManager for the remote case would make things clearer. The local case returns the full status inlined, not only its identifier.

Reply:
Mh okay then